docs/tutorials/assets-tokens/withdrawing-tokens.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Withdraw tokens
+sidebar_position: 5
+hide_title: false
+vega_network: TESTNET
+description: Withdraw staked or claimed tokens
+---
+
+## Check the status of your tokens
+
+How to withdraw your VEGA tokens depends on the status of the tokens. You can withdraw any tokens that are *vested*, or that you already hold on your Ethereum address.
```

Comment:
"on your ethereum address" reads weirdly to me but i also am unsure what's better. Needs to say address because it is key specific so wallet isn't as applicable. 

Maybe "or tokens owned by your ethereum address"?